examples/NonArduino/Pico/PicoHal.h

```diff
@@ -110,6 +151,19 @@ class PicoHal : public RadioLibHal {
     return (this->micros() - start);
   }
 
+  void tone(uint32_t pin, unsigned int frequency, unsigned long duration = 0) override {
+    // tones on the Pico are generated using bitbanging. This process is offloaded to the Pico's second core
+    multicore_reset_core1();
+    toneLoopPin = pin;
+    toneLoopFrequency = frequency;
+    toneLoopDuration = duration;
+    multicore_launch_core1(toneLoop);
+  }
+
+  void noTone(uint32_t pin) override {
+    multicore_reset_core1();
+  }
+
```

Comment:
Interesting imlementation, to offload this to the other core. However, I would add a large warning at the beginning of this file that this is being done, as I suspect most people will not expect it.

Reply:
Actually, you can't run this process on the one core if you're trying to do other tasks on it. There is a lot of jitter in the output that is caused. Thus, this process is offloaded to the other core, and jitter becomes unnoticeable and transmissions become much more consistent. Comment will be added.